distribution/packages/src/common/scripts/postinst

```diff
@@ -104,11 +104,18 @@ chmod g+s /etc/elasticsearch
 chmod 0750 /etc/elasticsearch
 
 if [ -f /etc/default/elasticsearch ]; then
-   chown root:elasticsearch /etc/default/elasticsearch
+    chown root:elasticsearch /etc/default/elasticsearch
 fi
 
 if [ -f /etc/sysconfig/elasticsearch ]; then
-   chown root:elasticsearch /etc/sysconfig/elasticsearch
+    chown root:elasticsearch /etc/sysconfig/elasticsearch
+fi
+
+if [ ! -f "$ES_PATH_CONF"/elasticsearch.keystore ]; then
+    /usr/share/elasticsearch/bin/elasticsearch-keystore create
+    chown root:elasticsearch "$ES_PATH_CONF"/elasticsearch.keystore
+    chmod 660 "$ES_PATH_CONF"/elasticsearch.keystore
+    md5sum "$ES_PATH_CONF"/elasticsearch.keystore > "$ES_PATH_CONF"/elasticsearch.keystore.md5sum
```

Comment:
Maybe name this something to indicate it is only the "initial" md5 sum? Otherwise someone might look at it and expect it to be kept up to date as changes are made to the keystone?

Reply:
It could also have "install" in the name instead of "initial" (or anything to indicate is is only for the version created at install time).

Reply:
I pushed 861d80fff08fe51de40df38a4ae27bb4ef64cd9e.